Dockerfile

```
@@ -0,0 +1,49 @@
+# Use Debian ARM64 (arm64v8) base image suitable for Apple Silicon
+FROM arm64v8/debian:bookworm-slim
+
+# Install build dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)